docs/CHANGELOG.rst

```diff
@@ -116,6 +116,9 @@ Changed
 * Set ``scheduling_class`` of gene and sample hierarchical clustering
   to ``interactive``
 * Set the cooksCutoff parameter to FALSE in ``deseq.R``
+* Bump STAR aligner version in ``resolwebio/rnaseq`` docker image to 2.5.4b
+* Remove sra-toolkit from ``resolwebio/rnaseq`` docker image
+
```

Comment:
Mention version bump of SRA Toolkit in ``resolwebio/utils`` Docker image. Remove the added empty line.

Reply:
The addition of sra-toolkit to `resolwebio/utils` already has its line in the unreleased section of the release notes. Possibly, only the specific version of the installed package can be mentioned there? Otherwise I omitted this on purpose.

Reply:
OK. Disregard my comment then.